client/my-sites/upgrades/checkout/domain-details-form.jsx

```diff
@@ -399,7 +403,7 @@ class DomainDetailsForm extends Component {
 	}
 
 	finish( options = {} ) {
-		this.setPrivacyProtectionSubscriptions( options.addPrivacy !== false );
+		this.setPrivacyProtectionSubscriptions( options.addPrivacy && options.addPrivacy !== false );
```

Comment:
If `options.addPrivacy` is truthy, it can never be `false`, right?

The old code here seems weird - if options.addPrivacy is `null` or `undefined` I wouldn't think we'd want to set the subscriptions. Was it supposed to be a string check or something?